### PR TITLE
verify glossary/tasks8&9

### PR DESCRIPTION
**Pull request by amyrchyk**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carina/demo/gui/components/GlossaryLinkItem.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    private List<ExtendedWebElement> titles;
+
+    private List<String> titlesLink = new ArrayList<>();
+
+    public GlossaryLinkItem(WebDriver driver, SearchContext searchContext) {
+	super(driver, searchContext);
+    }
+
+    public List<String> getTitles() {
+
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/components/GlossaryLinkItem.java:28]: remove space

File: src/test/java/com/qaprosoft/carina/demo/GsmArenaTests.java [hunk 2]
```diff
@@ -109,4 +110,28 @@ public void testVerifySearchigProcess() {
...
+    @Test(description = "JIRA#AUTO-0008")
+    @MethodOwner(owner = "amyrchyk")
+    @TestLabel(name = "feature", value = { "web", "regression" })
+    public void testVerifyGlossaryFirstLetterInTitle() {
+	HomePage homePage = new HomePage(getDriver());
+	homePage.open();
+	GlossaryPage glossaryPage = homePage.getFooterMenu().openGlossaryPage();
+	Assert.assertTrue(glossaryPage.isPageOpened(), "Glossary Page isn't opened");
+	Assert.assertTrue(glossaryPage.equalsCountTitleAndSectionsLinks(), "Count Element isn't equals");
+	Assert.assertTrue(glossaryPage.verifyLinkFirstLetterByTitle(), "The title isn't sorted by alphabet");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/GsmArenaTests.java:123]: change message

Comment (amyrchyk) [reply src/test/java/com/qaprosoft/carina/demo/GsmArenaTests.java:123]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/components/GlossaryLinkItem.java (hunk 1 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/components/GlossaryLinkItem.java:28]: done

Comment (vrudyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/components/GlossaryLinkItem.java:28]: I see space line 28
remove pls

Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/components/GlossaryLinkItem.java]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**